docs/pages/guides/editing-richtext.mdx

```diff
@@ -47,6 +47,7 @@ There are a couple of existing React Native libraries to allow rich-text editing
 
 - [`react-native-rich-editor`](https://github.com/wxik/react-native-rich-editor)
 - [`react-native-cn-quill`](https://github.com/imnapo/react-native-cn-quill)
+- [`@10play/tentap-editor`](https://github.com/10play/10Tap-Editor) - TenTap also gives you a way to customize the editor with some additional steps.
```

Comment:
```suggestion
- [`@10play/tentap-editor`](https://github.com/10play/10Tap-Editor)
```

We don't add descriptions to each library in this doc so let's follow the same pattern.